src/config/audio.rs

```diff
@@ -2,6 +2,26 @@ use serde::Deserialize;
 
 #[derive(Debug, Deserialize)]
 pub struct Audio {
+    #[serde(default = "Audio::default_latency_ms")]
+    pub latency_ms: u32,
     pub input: Option<String>,
     pub output: Option<String>,
 }
+
+impl Audio {
+    const DEFAULT_LATENCY_MS: u32 = 50;
+
+    fn default_latency_ms() -> u32 {
+        Self::DEFAULT_LATENCY_MS
+    }
+}
+
+impl Default for Audio {
```

Comment:
Is `Default` a standard(-library) trait? Who invokes it when?

Reply:
Yes, it's a standard library trait. The idea is that if a type has a reasonable default (the linter looks for a no-argument constructor), it should implement `Default`. `serde` can use it, and it also shows up in other places (eg. `Option::unwrap_or_default`). It seems reasonable to manually call `default` as well when that makes sense.